lib/membrane_element_portaudio/source.ex

```diff
@@ -7,6 +7,7 @@ defmodule Membrane.Element.PortAudio.Source do
   alias Membrane.Caps.Audio.Raw, as: Caps
```

Comment:
I would change order of directives here. I prefer to stick with order described in most style guides.
moduledoc > use > alias
(source https://github.com/christopheradams/elixir_style_guide#modules)

But since that's matter of taste I will leave that up to your conscience and judgement.